internal/btf/ext_info.go

```diff
@@ -328,6 +328,26 @@ type LineInfo struct {
 	LineCol     uint32
 }
 
+// FileName returns the file name as string
+func (li LineInfo) FileName(btfSpec *Spec) (string, error) {
+	return btfSpec.strings.Lookup(li.FileNameOff)
+}
+
+// Line returns the line of source code as a string
+func (li LineInfo) Line(btfSpec *Spec) (string, error) {
+	return btfSpec.strings.Lookup(li.LineOff)
+}
+
+// LineNum returns the line number within the FileName
+func (li LineInfo) LineNum() uint32 {
+	return li.LineCol >> 10
+}
+
+// ColNumber returns the column number within the Line
+func (li LineInfo) ColNum() uint32 {
+	return li.LineCol & 0x3ff
```

Comment:
The `0x3ff` magic constant should have a name.

Reply:
Agreed